datahub-graphql-core/src/main/resources/entity.graphql

```diff
@@ -704,17 +704,17 @@ type Dataset implements EntityWithRelationships & Entity {
     """
     An additional set of read only properties
     """
-    properties: DatasetProperties
+    properties(version: Long): DatasetProperties
 
     """
     An additional set of of read write properties
     """
-    editableProperties: DatasetEditableProperties
+    editableProperties(version: Long): DatasetEditableProperties
```

Comment:
Wow this is pretty cool... 

is this the aspect version or semantic version of the aspect?

Reply:
(guessing aspect version)

Reply:
Yeah it's just the aspect version, can't do semantic version until it's persistent